bin/crew

```diff
@@ -740,7 +777,7 @@ def unpack (meta)
       # Check the number of directories in the archive
       entries = Dir["#{@extract_dir}/*"]
       entries = Dir[@extract_dir] if entries.empty?
-      if entries.length == 0
+      if entries.length.zero?
```

Comment:
Would `if entries.empty?` work also?

Reply:
Yup. Is there any reason to use one over the other?

Reply:
rubocop suggested both, IIRC, and I just used one of them.

I'll be pretty pissed if neither is actually faster than `== 0`

Reply:
I imagine `.empty?` would be faster than `.length.zero?` by nanoseconds. :)

Reply:
Changed.